QueryBuilder_Neo4J/src/main/java/net/sf/esfinge/querybuilder/neo4j/Neo4JQueryParameters.java

```diff
@@ -0,0 +1,66 @@
+package net.sf.esfinge.querybuilder.neo4j;
+
+import org.neo4j.ogm.cypher.Filter;
+import org.neo4j.ogm.cypher.Filters;
+import org.neo4j.ogm.cypher.query.PagingAndSortingQuery;
+import org.neo4j.ogm.cypher.query.SortOrder;
+import org.neo4j.ogm.session.Neo4jSession;
+import org.neo4j.ogm.session.request.strategy.QueryStatements;
+
+public class Neo4JQueryParameters {
+
+	private String label;
+	private Filters filters;
+	private SortOrder sortOrder;
+
+	public Neo4JQueryParameters(String label, Filters filters, SortOrder sortOrder) {
+		this.label = label;
+		this.filters = filters;
+		this.sortOrder = sortOrder;
+	}
+	
+	public String getLabel() {
+		return label;
+	}
+	
+	public Filters getFilters() {
+		return filters;
+	}
+	
+	public SortOrder getSortOrder() {
+		return sortOrder;
+	}
+	
+	public String resolveQuery(Class<?> type, Neo4jSession session) {

```

Comment:
This method is coupled to Neo4jSession, PagingAndSortingQuery and QueryStatements, but this was the way I found to resolve the actual query that can be executed in Neo4J.